samples/Commerce/Commerce.UI/App.xaml

```diff
@@ -23,18 +23,18 @@
 				<ToolkitResources xmlns="using:Uno.Toolkit.UI" />
 
 				<!-- Load Material Toolkit resources -->
-				<MaterialToolkitResourcesV1 xmlns="using:Uno.Toolkit.UI.Material" />
+				<MaterialToolkitResources xmlns="using:Uno.Toolkit.UI.Material" />
```

Comment:
You'll need to change the reference of `MaterialResourcesV1` to `MaterialResources` at line 22